external/storm-hbase/src/main/java/org/apache/storm/hbase/bolt/AbstractHBaseBolt.java

```diff
@@ -37,8 +37,7 @@
 public abstract class AbstractHBaseBolt extends BaseRichBolt {
     private static final Logger LOG = LoggerFactory.getLogger(AbstractHBaseBolt.class);
 
-    protected OutputCollector collector;
-
+    protected transient OutputCollector collector;
```

Comment:
Why is this transient?  I don't see us accessing it from multiple threads, and especially not changing it on multiple threads?